public/javascripts/SVLabel/src/SVLabel/Storage.js

```diff
@@ -39,6 +39,10 @@ function TemporaryStorage(JSON, params) {
             set("completedOnboarding", null);
         }
 
+        if (!get("completedMissionAnonymously")){
```

Comment:
Signed in users already have first mission completion handled accordingly. Thus, I distinguished this storage value for anonymous users only.